pages/workouts/exercisesAdd/[id].js

```diff
@@ -27,6 +33,8 @@ const ExerciseAdd = ({ workouts, exercises, onAddExercise }) => {
         currentWorkout={currentWorkout}
         onAddExercise={onAddExercise}
         id={id}
+        exerciseAdded={exerciseAdded}
+        setExerciseAdded={setExerciseAdded}
```

Comment:
```suggestion
        updateAddedExercises={updateAddedExercises}
```
hier wird die neue Funktion durchgereicht